src/modules/rc_update/rc_update.cpp

```diff
@@ -602,16 +602,46 @@ void RCUpdate::UpdateManualSetpoint(const hrt_abstime &timestamp_sample)
 	_last_manual_control_setpoint_publish = manual_control_setpoint.timestamp;
 
 
-	// populate and publish actuator_controls_3 copied from mapped manual_control_setpoint
 	actuator_controls_s actuator_group_3{};
+	// copy in previous actuator control setpoint in case aux{1, 2, 3} isn't changed
+	_actuator_controls_3_sub.update(&actuator_group_3);
```

Comment:
It's almost always a bad idea to publish the same topic from multiple sources simultaneously.

Reply:
Hmm ok. I implemented the deconfliction logic that Beat recommended but I see how it could cause problems.

Reply:
And then you copy control group 6 here if published and changing and overwrite group 3? That would allow you to default the output of group 3 to the last active from either manual or external control. And someone just interested in external can map the mixer straight to group 6 and never have manual intervention.